### PR TITLE
Auto Translator Plugin Config accessible in android

### PR DESCRIPTION
**Pull request by PranitModak**
add configuration capability in android.
Moving config location to designated Melon directory

### CODE REVIEW

File: src/XUnity.AutoTranslator.Plugin.MelonMod/AutoTranslatorPlugin.cs [hunk 1]
```diff
@@ -19,18 +19,58 @@
...
       public override void OnApplicationLateStart()
       {
 #if IL2CPP
          var modFi = new FileInfo( Location );
          var gameDir = modFi.Directory.Parent;
          var unhollowedPath = Path.Combine( gameDir.FullName, "MelonLoader", "Il2CppAssemblies" );
          Il2CppProxyAssemblies.Location = unhollowedPath;
+         _dataFolder = Path.Combine( gameDir.FullName, "AutoTranslator" );
+         _configPath = Path.Combine( _dataFolder, "Config.ini" );
+         TranslationPath =_dataFolder;
```
Comment (ManlyMarco) [inline src/XUnity.AutoTranslator.Plugin.MelonMod/AutoTranslatorPlugin.cs:36]: Shouldn't this be done even outside of IL2CPP?

Comment (PranitModak) [reply src/XUnity.AutoTranslator.Plugin.MelonMod/AutoTranslatorPlugin.cs:36]: True. My bad I included it inside the il2cpp block

Comment (ManlyMarco) [reply src/XUnity.AutoTranslator.Plugin.MelonMod/AutoTranslatorPlugin.cs:37]: Now it doesn't compile, gameDir doesn't exist in non-il2cpp.

Comment (PranitModak) [reply src/XUnity.AutoTranslator.Plugin.MelonMod/AutoTranslatorPlugin.cs]: Build tested.

File: src/XUnity.AutoTranslator.Plugin.MelonMod/AutoTranslatorPlugin.cs [hunk 2]
```diff
@@ -29,8 +32,47 @@ public override void OnApplicationLateStart()
          var unhollowedPath = Path.Combine( gameDir.FullName, "MelonLoader", "Il2CppAssemblies" );
          Il2CppProxyAssemblies.Location = unhollowedPath;
 #endif
+         var gameDirectory = MelonUtils.GameDirectory;
+         LoggerInstance.Msg( "Mod Directory : "+ gameDirectory );
```
Comment (ManlyMarco) [inline src/XUnity.AutoTranslator.Plugin.MelonMod/AutoTranslatorPlugin.cs:36]:
```suggestion
```

**Review by ManlyMarco — APPROVED** [review]
Looks good.

**Pull request merged**